### ISSUE TITLE
Tag renderer can throw exceptions

### ISSUE THREAD

**Issue by Bios-Marcel** [Bios-Marcel/baka#16]
Probably happens for classifications without a tag:

```
java.lang.NullPointerException
  at java.base/java.util.Objects.requireNonNull(Objects.java:220)
  at java.base/java.util.Optional.of(Optional.java:113)
  at java.base/java.util.stream.ReduceOps$2ReducingSink.get(ReduceOps.java:129)
  at java.base/java.util.stream.ReduceOps$2ReducingSink.get(ReduceOps.java:107)
  at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
  at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
  at java.base/java.util.stream.ReferencePipeline.reduce(ReferencePipeline.java:677)
  at link.biosmarcel.baka/link.biosmarcel.baka.view.PaymentFX.lambda$new$2(PaymentFX.java:39)
```

**Comment by Bios-Marcel** [comment]
![java_PG4EEvBvbj](https://github.com/user-attachments/assets/844cbc98-84d8-4b97-8d07-fd8f49f62731)

Generating data over 10 years, this actually looks horrible. We somehow need to calculate some type of precision and an x-axis value that isn't purely numeric.

**Comment by Bios-Marcel** [comment]
This will potentially be fixed automatically as soon as #11 is implemented.

**Comment by Bios-Marcel** [comment]
Tags simply won't be nullable anymore after the merge of #20
